packages/react/src/components/dialog/Dialog.module.scss

```diff
@@ -0,0 +1,71 @@
+@import '../../styles/common';
```

Comment:
The file inside styles folder seems to be named with underscore prefix: _common. But the import seems to work also without the underscore?

Reply:
Good catch 👍
Hmm.., I think we should use the underscore name. IIRC, the underscore tells the SASS that there is no need to generate a new CSS file. But this must be confirmed.